saleor/graphql/schema.graphql

```diff
@@ -5799,6 +5799,7 @@ enum WebhookEventTypeEnum {
   CHECKOUT_CREATED
   CHECKOUT_UPDATED
   FULFILLMENT_CREATED
+  NOTIFY
```

Comment:
The name should follow the convention used for other webhooks, so I would suggest:
- `NOTIFICATION_SENT`
- `USER_NOTIFICATION_SENT`